.gitignore

```diff
@@ -35,4 +35,3 @@ composer/
 nosetests.xml
 virtenv
 .vscode
-test_results
```

Comment:
why are you removing this ?

Reply:
ohh l think l removed it unknowingly:smile:  l will add it then push again

Reply:
Done with this